packages/build/src/commands/core_command.js

```diff
@@ -13,9 +13,19 @@ const fireCoreCommand = async function ({
   logs,
   childEnv,
   featureFlags,
+  netlifyConfig,
```

Comment:
Passing `netlifyConfig` down the chain so that it reaches `packages/build/src/plugins_core/functions/index.js`.